app/models/queries/projects.rb

```diff
@@ -72,5 +74,6 @@ module Queries::Projects
     select Selects::ProjectPhase
     select Selects::RequiredDiskSpace
     select Selects::Status
+    select Selects::UpdatedAt
```

Comment:
❓ So... you can `select` and `orderBy` for `updatedAt`, too, now?

Reply:
That is correct. Those are visible in the UI and @ulferts  and I agreed that it makes sense to offer `updated_at` for all three categories if we offer it for one.